sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcUtil.java

```diff
@@ -252,4 +262,67 @@ private static Calendar withTimestampAndTimezone(DateTime dateTime) {
 
     return calendar;
   }
+
+  /** Create partitions on a table. */
```

Comment:
It would be useful to add a javadoc with examples of different cases.

Reply:
There are few examples in the header of the JdbcIO.java file, do I need to add more?